comp/core/gui/guiimpl/checks.go

```diff
@@ -56,11 +49,8 @@ func getFleetPoliciesPath() string {
 }
 
 // Adds the specific handlers for /checks/ endpoints
-func checkHandler(r *mux.Router, collector collector.Component, ac autodiscovery.Component) {
+func checkHandler(r *mux.Router) {
 	r.HandleFunc("/running", http.HandlerFunc(sendRunningChecks)).Methods("POST")
-	r.HandleFunc("/run/{name}", http.HandlerFunc(runCheckHandler(collector, ac))).Methods("POST")
```

Comment:
Nothing else uses these endpoints? They were only being called by the GUI's js code?

Reply:
Correct. The GUI has its server, and these handlers were only used with the JS functions I remove